services/sentry/eventAggregator.js

```diff
@@ -64,6 +72,14 @@ function makeRecorder(channelId) {
 			return hasAccess
 		}
 
+		const priceRuleModifyEvs = events.filter(x => x.type === eventTypes.update_price_rules)
```

Comment:
you can use `find` here rather than relying on `filter` and then getting the last element

Reply:
Find will return the first item in the list it finds